pkg/event/target/mqtt.go

```diff
@@ -173,7 +147,8 @@ func NewMQTTTarget(id string, args MQTTArgs) (*MQTTTarget, error) {
 	var store Store
 
 	if args.QueueDir != "" {
-		store = NewQueueStore(args.QueueDir, args.QueueLimit)
+		queueDir := filepath.Join(args.QueueDir, storePrefix+"-mqtt-"+id)
```

Comment:
Directory format here becomes `<queueDir>/minio-mqtt-1` instead of `<queueDir>/minio-<queue-ARN>/` (as mentioned in https://github.com/minio/minio/pull/7343#issuecomment-474300384)

Reply:
The `queue-ARN` can be neither accessible nor constructed in the event package as it is under cmd package. So i decided to keep it this way. which is still unique. Let me re-think on this.